static/app/components/events/featureFlags/featureFlagOnboardingSidebar.tsx

```diff
@@ -0,0 +1,399 @@
+              }
+              triggerProps={{'aria-label': currentProject?.slug}}
+              options={projectSelectOptions}
+              position="bottom-end"
+            />
+          </div>
+        </HeaderActions>
+        <OnboardingContent currentProject={selectedProject} hasDocs={hasDocs} />
+      </TaskList>
+    </TaskSidebarPanel>
+  );
+}
+
+function OnboardingContent({
+  currentProject,
+  hasDocs,
+}: {
+  currentProject: Project;
+  hasDocs: boolean;
+}) {
+  const organization = useOrganization();
+  const openFeatureProviders = [ProviderOptions.LAUNCHDARKLY];
+  const sdkProviders = [ProviderOptions.LAUNCHDARKLY];
+
+  // First dropdown: OpenFeature providers
+  const openFeatureProviderOptions = openFeatureProviders.map(provider => {
+    return {
+      value: provider,
+      textValue: provider,
+      label: <TextOverflow>{provider}</TextOverflow>,
+    };
+  });
+
+  const [openFeatureProvider, setOpenFeatureProvider] = useState<{
+    value: string;
+    label?: ReactNode;
+    textValue?: string;
+  }>(openFeatureProviderOptions[0]);
+
+  // Second dropdown: other SDK providers
+  const sdkProviderOptions = sdkProviders.map(provider => {
+    return {
+      value: provider,
+      textValue: provider,
+      label: <TextOverflow>{provider}</TextOverflow>,
+    };
+  });
+
+  const [sdkProvider, setsdkProvider] = useState<{
+    value: string;
+    label?: ReactNode;
+    textValue?: string;
+  }>(sdkProviderOptions[0]);
+
+  const defaultTab: string = 'openFeature';
+  const {getParamValue: setupMode, setParamValue: setSetupMode} = useUrlParams(
+    'mode',
+    defaultTab
+  );
+
+  const currentPlatform = currentProject.platform
+    ? platforms.find(p => p.id === currentProject.platform) ?? otherPlatform
+    : otherPlatform;
+
+  const {
+    docs,
+    dsn,
+    isLoading: isProjKeysLoading,
+    projectKeyId,
+  } = useLoadGettingStarted({
+    platform: currentPlatform,
+    projSlug: currentProject.slug,
+    orgSlug: organization.slug,
+    productType: 'featureFlags',
+  });
+
+  const radioButtons = (
+    <Header>
+      <StyledRadioGroup
+        label="mode"
+        choices={[
+          [
+            'openFeature',
+            <PlatformSelect key="platform-select">
+              {tct(
+                'I use the [link:OpenFeature] SDK using a provider from [providerSelect]',
+                {
+                  providerSelect: (
+                    <CompactSelect
+                      triggerLabel={openFeatureProvider.label}
+                      value={openFeatureProvider.value}
+                      onChange={setOpenFeatureProvider}
+                      options={openFeatureProviderOptions}
+                      position="bottom-end"
+                      key={openFeatureProvider.textValue}
+                      disabled={setupMode() === 'other'}
+                    />
+                  ),
+                  link: <ExternalLink href="" />, // TODO: link
```

Comment:
wasn't sure whether to use the OpenFeature site here (https://openfeature.dev/) or the sentry specific docs (https://docs.sentry.io/platforms/python/integrations/openfeature/)